Patterns/Swedish/Swedish-TimeZone.yaml

```diff
@@ -348,6 +645,101 @@ AmbiguousTimezoneList: !list
 AbbrToMinMapping: !dictionary
  types: [ string , int ]
  entries:
+   # Swedish extensions
+   acwst: 525
+   almt: 360
+   #amt: 240
+   #amt: -240
+   anat: 720
+   aqtt: 300
+   #ast: 180
+   #ast: -240
```

Comment:
We're still missing the -1000 resolutions for the ambiguous timezone names. Can you add thise? Then I think we can merge. Thanks!

Reply:
Yeah, I'll fix this. I just wanted to commit my changes to see that it all worked before I refine the patterns.

Reply:
@pete1854 Sorry for the repetition. I saw your comment that you had implemented one of the required changes and as the build was passing I assumed the PR was ready for re-review.
What about from now on you leave a comment when the PR is ready to review after changes? Thanks for the great work BTW.